server/src/cache.ts

```diff
@@ -5,6 +5,7 @@ import { REDIS_URL } from './constants'
 
 let rcache: RedisClient
 let get: (arg: string) => Promise<string>
+let del: (arg: string) => Promise<number>
```

Comment:
You finally didn't use this, did you?

Reply:
no I didn't but now it's there when we need it :)

Reply:
What about YAGNI? ;)

Reply:
what do you think the french government thinks about YAGNI now that we're out of masks and respirators ?

Reply:
covid-19 should be an inspiration to all developers around the world

Reply:
❤️